model/riscv_insts_fext.sail

```diff
@@ -274,8 +278,8 @@ union clause ast = LOAD_FP : (bits(12), regidx, regidx, word_width)
 
 /* AST <-> Binary encoding ================================ */
 
-mapping clause encdec = LOAD_FP(imm, rs1, rd, HALF)          if extensionEnabled(Ext_Zfh)
-                    <-> imm @ rs1 @ 0b001 @ rd @ 0b000_0111  if extensionEnabled(Ext_Zfh)
+mapping clause encdec = LOAD_FP(imm, rs1, rd, HALF)          if extensionEnabled(Ext_Zfhmin)
```

Comment:
Yeah. I went back and forth on this, but decided that since Zfh implies Zfhmin this was cleaner. Could definitely go the other route if you think that’s clearer though.

Reply:
Let's stick with this.